Sources/SpeziOpenAI/MessageView.swift

```diff
@@ -82,12 +79,13 @@ struct MessageView_Previews: PreviewProvider {
     static var previews: some View {
         ScrollView {
             VStack {
-                MessageView(Chat(role: .system, content: "System Message!"), hideSystemMessages: false)
-                MessageView(Chat(role: .system, content: "System Message (hidden)!"))
-                MessageView(Chat(role: .user, content: "User Message!"))
-                MessageView(Chat(role: .assistant, content: "Assistant Message!"))
+                MessageView(Chat(role: .system, content: "System Message!"), hideMessagesWithRoles: [.system])
+                MessageView(Chat(role: .system, content: "System Message (hidden)!"), hideMessagesWithRoles: [.system])
+                MessageView(Chat(role: .function, content: "Function Message!"), hideMessagesWithRoles: [.function])
+                MessageView(Chat(role: .user, content: "User Message!"), hideMessagesWithRoles: [.user])
+                MessageView(Chat(role: .assistant, content: "User Message!"), hideMessagesWithRoles: [.user])
```

Comment:
If you check out the preview, it seems like all of the messages are hidden (except the .assistant message).
Something like this might make more sense and allows you to visually test the behaviour:
```suggestion
                MessageView(Chat(role: .system, content: "System Message!"), hideMessagesWithRoles: [])
                MessageView(Chat(role: .system, content: "System Message (hidden)!"))
                MessageView(Chat(role: .function, content: "Function Message!"), hideMessagesWithRoles: [.system])
                MessageView(Chat(role: .user, content: "User Message!"))
                MessageView(Chat(role: .assistant, content: "Assistant Message!"))
```